Systems/engine.xml

```diff
@@ -609,6 +609,52 @@
         <output>
             <property>/fdm/jsbsim/propulsion/engine[1]/volumetric-efficiency</property>
         </output>
-    </filter>            
+    </filter>
+
+    <!-- ============================================================== -->
+    <!-- Complex engine procedures                                      -->
+    <!-- ============================================================== -->
+
+    <logic>
+        <name>Complex engine procedures => oil consumption allowed</name>
```

Comment:
I didn't see your comment (seems I need to press F5 for these things), so it should be fixed in a next PR.